recipe/meta.yaml

```diff
@@ -40,8 +41,6 @@ test:
     - urllib3
```

Comment:
You can change bortlipy to `brotli-python >=1.0.9` and remove the corresponding comment.